src/components/shared/FormattedDateRangeText/FormattedDateRangeText.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FormattedMessage, FormattedDate } from 'react-intl';
+import { addDays } from '../../../utils';
+
+export const FormattedDateRangeText = ({ dateFrom, dateTo, today }) => {
+  return (
+    <div>
+      {today ? (
+        <FormattedDate value={dateTo} />
```

Comment:
Maybe it should be _dateFrom_ (which is a pure date)

```suggestion
        <FormattedDate value={dateFrom} />
```

Reply:
fixed